changelogs/current.yaml

```diff
@@ -124,6 +124,9 @@ removed_config_or_runtime:
 - area: upstream
   change: |
     Removed runtime flag ``envoy.reloadable_features.exclude_host_in_eds_status_draining``.
+- area: http
+  change: |
+    Set ``envoy.reloadable_features.streaming_shadow`` to default true.
```

Comment:
this section is for removed
I think you want to add this to the behavioral changes / feature section , explain that streaming shadows are supported, and that you can revert to prior behavior by flipping the flag false.
/wait